source/modules/assets/computers.rst

```diff
@@ -1,6 +1,135 @@
+
+.. image:: images/computer-view.png
+   :alt: module assets - computer
+   :align: center
+   :scale: 31 %
+
+
+* Name
+* `Location <../../common_fields.html#location>`_
+* `Technician in charge <../../common_fields.html#technician-in-charge>`_
+* `Group in charge <../../common_fields.html#group-in-charge>`_
+* `Alternate usernmame number <../../common_fields.html#Alternate-usernmame-number>`_
+* `Alternate usernmame <../../common_fields.html#Alternate-usernmame-number>`_
+* `User <../../common_fields.html#user>`_
+* `Group <../../common_fields.html#group>`_
+* `Comments <../../common_fields.html#comments>`_
+* `Status <../../common_fields.html#status>`_
+* `Computer type <../../common_fields.html#computer-type>`_
+* `Manufacturer <../../common_fields.html#manufacturer>`_
+* `Model <../../common_fields.html#model>`_
+* `Serial number <../../common_fields.html#serial-number>`_
+* `Inventory number <../../common_fields.html#inventory-number>`_
+* `Network <../../common_fields.html#network>`_
+* `UUID <../../common_fields.html#uuid>`_
+* `Update source <../../common_fields.html#update-source>`_
+
+If your computer has been inventoried by the automatic inventory, you can find informtions about the `Agent GLPI <https://glpi-agent.readthedocs.io/en/latest/>`_
+
+* Agents
+* Public contact address
+* Agents Status
+* Useragent
+* Last contact
+* Request inventory
+* Inventory tag
+* Last inventory update
+
+Impact Analysis
+---------------
+
+`Impact analysis <../../common_tabs.html#impact-analysis>`_ enables an infrastructure diagram to be drawn up, showing the dependencies and impacts in the event of equipment loss.
+This can be saved and exported
+
+
+Operating system
+----------------
+
+Operating system includes information about your machine's OS :
+
+* Name
+* Architecture
+* Kernel
+* Product ID
+* Company
+* Installation date
+* Version
+* Service pack
+* Edition
+* Serial number
+* Owner
+* Host ID
+
+
+Components
+----------
+
+This tab lists the PC's components :
+
+* BIOS
+* Processor
+* Memory
+* Hard Drive
+* Network card
+* Drive
+* Battery
+* Graphics card
+* Soundcard
+* Controller
+
+Each item has its own information (name, model, brand, memory capacity, number of cores/threads, etc.).
+
+Lines
+-----
+
+You can add telephone lines created in :doc:`Lines <../management/lines>`
+
+
+Volumes
+-------
+
+Summarises all the volumes present (hard disk, DVD) as well as the partitions present on the workstation
+(virtual disks such as Google Cloud may appear if they are installed as a network drive).
+
+* Name
+* Automatic inventory (Yes /No)
+* partition
+* Mount point
+* File system
+* Global size
+* Free size
+* Free percentage
+* Encryptation (if the disk is encrypted, a padlock will be displayed)
+
+
+Software
+--------
+
+Lists all the applications brought up during the inventory and those added manually
+
+
+
+
+
+
+**-------------------OLD DOCUMENTATION-----------------**
```

Comment:
The whole "old documentation" should certainly be removed

Reply:
Yes, of course, but I don't want to delete anything until I'm sure that all the information has been transferred elsewhere.